package-lock.json

```diff
@@ -2646,9 +2646,9 @@
       }
```

Comment:
Please, remove from commit changes to `package-lock.json`, it can create unnecessary issues

Reply:
fixed